src/media/AvatarMember.jsx

```diff
@@ -30,16 +31,22 @@ class AvatarMember extends React.PureComponent {
 			className
 		);
 
-		const noPhotoImage = require('base64-image-loader!swarm-icons/dist/optimized/profile.svg');
+		const noPhotoIcon = (<Icon
+			shape="profile"
+			size="m"
+		/>);
 
-		return (
-			<Avatar
-				alt={member.name}
-				src={showNoPhoto ? noPhotoImage : member.photo[photoLink]}
-				className={classNames}
-				{...other}
-			/>
-		);
+		const allProps = {
+			alt: member.name,
+			src: !showNoPhoto && member.photo[photoLink],
+			className: classNames,
+		};
+
+		if (showNoPhoto) {
+			return <Avatar {...allProps} {...other}>{noPhotoIcon}</Avatar>;
```

Comment:
minor, but instead of 2 return statements, I would recommend adding `children: showNoPhoto && noPhotoIcon` as part of `allProps` and just having a single return statement